### ISSUE TITLE
Upgrade the CNN for better results 🖼️

### ISSUE THREAD

**Issue by BlueBlaze6335** [gdscjgec/ML-Gallery#3]
## 🖼️ Update the CNN by adding layers !! 

**You can add Convolution layers and edit the CNN to generate better results !!** 

The accuracy is very poor. You can add layers and make changes and tweak the model to generate better results. You can create a new model named "Net-X" , where X is the network number and also plot the training and validation accuracy of your model !! And when you are done update the readme with the results and also the model architecture. 

**Additional**
You can add Validation loss vs. training loss graph and  confusion matrix.

**Comment by Tanmoydey21** [comment]
Please assign me this issue so that I can update the CNN.
Thank you.

**Comment by Tanmoydey21** [comment]
> ## 🖼️ Update the CNN by adding layers !!
> **You can add Convolution layers and edit the [CNN](https://github.com/gdscjgec/ML-Gallery/tree/main/Convolutional%20Neural%20Network) to generate better results !!**
> 
> The accuracy is very poor. You can add layers and make changes and tweak the model to generate better results. You can create a new model named "Net-X" , where X is the network number and also plot the training and validation accuracy of your model !! And when you are done update the readme with the results and also the model architecture.
> 
> **Additional** You can add Validation loss vs. training loss graph and confusion matrix.

Please assign me this issue so that I can update the CNN.
Thank you.

**Comment by BlueBlaze6335** [comment]
@Tanmoydey21 I have assigned you this issue !! If you need any help , i guess you know how to reach me. Otherwise you can ask questions in the GDSC JGEC Discord server - [Link to text channel](https://discord.gg/ZZpUQFXUBZ)

**Comment by BlueBlaze6335** [comment]
Everyone please check the discussion section !!
